impl/src/test/groovy/io/jsonwebtoken/impl/io/RuntimeClasspathSerializerLocatorTest.groovy

```diff
@@ -98,36 +109,27 @@ class RuntimeClasspathSerializerLocatorTest {
 
     @Test
     void testOrgJson() {
-        def locator = new RuntimeClasspathSerializerLocator() {
-            @Override
-            protected boolean isAvailable(String fqcn) {
-                if (JacksonSerializer.class.getName().equals(fqcn)) {
-                    return false //skip it to allow the OrgJson impl to be created
-                }
-                return super.isAvailable(fqcn)
-            }
-        }
+        prepareFakeServiceClassLoader(TEST_SERVICE_DESCRIPTOR_ORGJSON)
 
-        def serializer = locator.getInstance()
+        def serializer = new RuntimeClasspathSerializerLocator().getInstance()
         assertTrue serializer instanceof OrgJsonSerializer
     }
-    
+
     @Test
     void testGson() {
-        def locator = new RuntimeClasspathSerializerLocator() {
-            @Override
-            protected boolean isAvailable(String fqcn) {
-                if (JacksonSerializer.class.getName().equals(fqcn)) {
-                    return false //skip it to allow the Gson impl to be created
-                }
-                if (OrgJsonSerializer.class.getName().equals(fqcn)) {
-                    return false //skip it to allow the Gson impl to be created
-                }
-                return super.isAvailable(fqcn)
-            }
-        }
+        prepareFakeServiceClassLoader(TEST_SERVICE_DESCRIPTOR_GSON)
 
-        def serializer = locator.getInstance()
+        def serializer = new RuntimeClasspathSerializerLocator().getInstance()
         assertTrue serializer instanceof GsonSerializer
     }
+
+    private void prepareNoServiceDescriptorClassLoader() {
```

Comment:
doh, i missed this class, i need to clean this test up a little to make it look like the Deserializer equivalent